man/man8/radcrypt.8

```diff
@@ -20,6 +23,16 @@ In this case \fIhashed_password\fP will be checked to see if it matches
 status will be 1, otherwise "Password BAD" will be printed and exit status
 will be 0 (Note this is the opposite of a normal successful shell status).
 
+That said, it should be noted that the current (August 2017) best practice is to 
+not use or store passwords at all. Rather, if possible, administrators are encouraged 
+to get rid of passwords and use X509 certs instead. Supplicant behaviour is poor on 
+macOS/iOS and Windows. If you use EAP-TTLS, EAP-PEAP etc... anyone sufficiently motivated 
+will be able to harvest your user's passwords.
+
+However, if generating high quality hashed passwords is a priority, dovecot's doveadm
+utility is quite useful (e.g. doveadm pw -s SSHA512) or openssl (echo -n "pw" | 
+openssl dgst -sha512).
+
```

Comment:
You're right, that makes sense. In reading a bit more I realize that "echo -n 'password' | openssl dgst -sha512" isn't really good advice because there's no salt. So unless you know how to cook up that command properly, perhaps it's best left to the imagination.

As for the rest, these changes look fabulous.